io.github.hakandundar34coding.system-monitoring-center.yml

```diff
@@ -0,0 +1,90 @@
+app-id: io.github.hakandundar34coding.system-monitoring-center
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: system-monitoring-center
+
+finish-args:
+  # For X11 shared memory access (higher performance for X11).
+  - --share=ipc
+  # For X11 access.
+  - --socket=x11
+  # For Wayland access.
+  - --socket=wayland
+  # For X11 fallback.
+  - --socket=fallback-x11
+  # For host OS commands access. Otherwise it gives this error: Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
+  - --socket=session-bus
+  # For monitoring data download/upload speed on network cards.
+  - --device=all
+  # For OpenGL access (hardware acceleration).
+  - --device=dri
+  # For monitoring data download/upload speed on network cards.
+  - --share=network
+  # For host OS commands access (such as 'systemctl' for service details, 'ls /proc' for process list, 'ps' for process information, etc.).
+  - --talk-name=org.freedesktop.Flatpak
+  # For reading several files (systemd service files, process information in '/proc' folder, etc.) of host OS.
+  - --filesystem=host:ro
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/man
+
+modules:
+
+  # For using hardware database of "udev".
+  - name: eudev
+    cleanup:
+      - /bin
+      - /sbin
+      - /include
+      - /lib*/udev
+      - /lib/pkgconfig
+    buildsystem: autotools
+    post-install:
+      - udevadm hwdb --update
+    sources:
+      - type: archive
+        url: https://github.com/eudev-project/eudev/releases/download/v3.2.11/eudev-3.2.11.tar.gz
+        sha256: 19847cafec67897da855fde56f9dc7d92e21c50e450aa79068a7e704ed44558b
+        x-checker-data:
+          type: anitya
+          project-id: 13466
+          stable-only: true
+          url-template: https://github.com/eudev-project/eudev/releases/download/v$version/eudev-$version.tar.gz
+
+  # For getting IPv4 and IPv6 addresses by using 'ip a show [network_card]' command.
+  - name: iproute2
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/shemminger/iproute2/archive/refs/tags/v5.11.0.tar.gz
+        sha256: 16b79e6ce65d4d5fd425cef2fd92a58c403a93faeeed0e0a3202b36a8e857d1f
+    build-commands:
+      - ./configure --prefix=/
```

Comment:
prefix = `/` is likely wrong. Aslo you should be using the autotools build system.

Reply:
how about this? ^^^^^

Reply:
I changed it but it did not work. Also another application on Flatpak used the same prefix.

Reply:
I changed the build system but it did not work.

Reply:
I truly cannot gues the errors.